extensions/http-status-codes/src/codes.json

```diff
@@ -0,0 +1,322 @@
+  },
+  {
+    "code": "415",
+    "description": "Unsupported Media Type",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.16]"
+  },
+  {
+    "code": "416",
+    "description": "Range Not Satisfiable",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.17]"
+  },
+  {
+    "code": "417",
+    "description": "Expectation Failed",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.18]"
+  },
+  {
+    "code": "418",
+    "description": "I'm a teapot",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.19]"
+  },
+  {
+    "code": "420",
+    "description": "Nice",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.69]"
+  },
+  {
+    "code": "421",
+    "description": "Misdirected Request",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.20]"
+  },
+  {
+    "code": "422",
+    "description": "Unprocessable Content",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.21]"
+  },
+  {
+    "code": "423",
+    "description": "Locked",
+    "reference": "[RFC4918]"
+  },
+  {
+    "code": "424",
+    "description": "Failed Dependency",
+    "reference": "[RFC4918]"
+  },
+  {
+    "code": "425",
+    "description": "Too Early",
+    "reference": "[RFC8470]"
+  },
+  {
+    "code": "426",
+    "description": "Upgrade Required",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.5.22]"
+  },
+  {
+    "code": "428",
+    "description": "Precondition Required",
+    "reference": "[RFC6585]"
+  },
+  {
+    "code": "429",
+    "description": "Too Many Requests",
+    "reference": "[RFC6585]"
+  },
+  {
+    "code": "431",
+    "description": "Request Header Fields Too Large",
+    "reference": "[RFC6585]"
+  },
+  {
+    "code": "451",
+    "description": "Unavailable For Legal Reasons",
+    "reference": "[RFC7725]"
+  },
+  {
+    "code": "500",
+    "description": "Internal Server Error",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.6.1]"
+  },
+  {
+    "code": "501",
+    "description": "Not Implemented",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.6.2]"
+  },
+  {
+    "code": "502",
+    "description": "Bad Gateway",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.6.3]"
+  },
+  {
+    "code": "503",
+    "description": "Service Unavailable",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.6.4]"
+  },
+  {
+    "code": "504",
+    "description": "Gateway Timeout",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.6.5]"
+  },
+  {
+    "code": "505",
+    "description": "HTTP Version Not Supported",
+    "reference": "[RFC-ietf-httpbis-semantics, Section 15.6.6]"
+  },
+  {
+    "code": "506",
+    "description": "Variant Also Negotiates",
+    "reference": "[RFC2295]"
+  },
+  {
+    "code": "507",
+    "description": "Insufficient Storage",
+    "reference": "[RFC4918]"
+  },
+  {
+    "code": "508",
+    "description": "Loop Detected",
+    "reference": "[RFC5842]"
+  },
```

Comment:
`509 - Bandwidth Limit Exceeded` is missing 👀

Just an opinion, not sure how useful is the `reference` (not even copy-able). _Maybe_ this list could be self-maintainable if using `require("http").STATUS_CODES` as source instead. Since Raycast comes with Node.js built-in, it means every Node.js update _may_ include newer HTTP status codes and this extension won't need to be updated.

Reply:
>Just an opinion, not sure how useful is the reference (not even copy-able). 

Fair enough, I had it in the data set so thought that I might as well show it. But I agree: no added value. I'll get rid of it.